modules/my_health/spec/request/v1/messages_request_spec.rb

```diff
@@ -9,327 +9,377 @@
   include SchemaMatchers
 
   let(:user_id) { '10616687' }
+  let(:mhv_account_type) { 'Premium' }
   let(:inbox_id) { 0 }
   let(:message_id) { 573_059 }
   let(:va_patient) { true }
   let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
   let(:inflection_header) { { 'X-Key-Inflection' => 'camel' } }
 
   before do
-    allow(SM::Client).to receive(:new).and_return(authenticated_client)
     sign_in_as(current_user)
   end
 
-  context 'Basic User' do
-    let(:mhv_account_type) { 'Basic' }
-
-    before { get '/my_health/v1/messaging/messages/categories' }
-
-    include_examples 'for user account level', message: 'You do not have access to messaging'
-    include_examples 'for non va patient user', authorized: false, message: 'You do not have access to messaging'
-  end
+  context 'when sm session policy is enabled' do
+    before do
+      Flipper.enable(:mhv_sm_session_policy)
```

Comment:
same here, is you allow(Flipper)... you won't have the possibility of running into flaky specs